static/src/javascripts/projects/commercial/modules/IpsosTaggingScript.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+import { onConsentChange, getConsentFor } from '@guardian/consent-management-platform';
+import config from 'lib/config';
+
+let dm;
+let DotMetricsObj;
+const IpsosTagging = function () {
+
+    console.log("Ipsos tag fired");
+    window.dm = window.dm ||{ AjaxData:[]};
+    window.dm.AjaxEvent = function(et, d, ssid, ad){
+        dm.AjaxData.push({ et,d,ssid,ad});
+        // eslint-disable-next-line no-unused-expressions
+        window.DotMetricsObj && DotMetricsObj.onAjaxDataUpdate();
```

Comment:
if eslint is complaining, you're meant to conform to what it asks :)

```suggestion
        if (window.DotMetricsObj) {
                DotMetricsObj.onAjaxDataUpdate();
        }
```

Reply:
This was part of the script that Ipsos Mori sent so was a bit worried about changing it! I'll change it over to this 👍